File: hackerrank/LinkedIn/Consecutive1's/ConsecutiveOnes.py
num = bin(int(input().strip()))[2:]
c = 0;prev = 0
for i in range(len(num)):
	if num[i] is '1':
		c+=1
		prev = c if c>= prev else prev
	elif num[i] is '0':
		c = 0
print(prev)

File: hackerrank/LinkedIn/Consecutive1's/ConsecutiveOnes.java
import java.util.*;
public class ConsecutiveOnes {
	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		int num = Integer.parseInt(sc.nextLine().trim());
		String str = Integer.toBinaryString(num);
		System.out.println(Integer.toBinaryString(num));
		int count = 0, prev = 0;
		for (int i = 0; i < str.length(); i ++) {
			if (str.charAt(i) =='1') {
				count++;
				if (count >= prev){
					prev = count;
				}
			} else if (str.charAt(i)=='0') {
				count=0;
			}
		}
		System.out.println(prev);
	}
}